frontend/src/framework/internal/components/SelectEnsemblesDialog/selectEnsemblesDialog.tsx

```diff
@@ -241,7 +311,7 @@ export const SelectEnsemblesDialog: React.FC<SelectEnsemblesDialogProps> = (prop
                     </div>
                     <div className="flex flex-col flex-grow gap-4 p-4">
                         <Label text="Selected Ensembles">
-                            <table className="w-full border border-collapse table-fixed">
+                            <table className="w-full border border-collapse table-fixed  text-sm">
```

Comment:
```suggestion
                            <table className="w-full border border-collapse table-fixed text-sm">
```
When thinking about it, should we introduce https://tailwindcss.com/blog/automatic-class-sorting-with-prettier (in a separate PR/issue)?